implementation/ruleset.json

```diff
@@ -0,0 +1,3 @@
+[
```

Comment:
explain

Reply:
Discussed with Cfir, should remain as is.